modules/installation-adding-registry-pull-secret.adoc

```diff
@@ -96,14 +96,41 @@ The contents of the file resemble the following example:
 ----
 // An additional step for following this procedure when using oc-mirror as part of the disconnected install process.
 ifdef::oc-mirror[]
-. Save the file either as `~/.docker/config.json` or `$XDG_RUNTIME_DIR/containers/auth.json`.
+. Save the file either as `~/.docker/config.json` or `$XDG_RUNTIME_DIR/containers/auth.json`. 
```

Comment:
Hi Steven, should we add same steps for $XDG_RUNTIME_DIR/containers too ?

Reply:
@xiuwang That makes sense to me. I wasn't sure if `$XDG_RUNTIME_DIR/containers` was automatically created or if it was manually created. I can add the creation of that folder now. Thanks